ledger/ledger-consensual/src/main/kotlin/net/corda/v5/ledger/consensual/transaction/ConsensualTransactionBuilder.kt

```diff
@@ -72,5 +73,6 @@ interface ConsensualTransactionBuilder {
      *      unintentional duplicate transactions.
      */
     @Suspendable
-    fun sign(vararg signatories: PublicKey): ConsensualSignedTransaction
+    @Deprecated("Temporary function until the argumentless version gets available")
+    fun toSignedTransaction(vararg signatories: PublicKey): ConsensualSignedTransaction
```

Comment:
If we're going to remove this and its only us using them atm, then we can remove the `varargs` implementation as its not really useful to us.

Reply:
I have removed it.